extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-hive-metastore-service-chart/README.md

```diff
@@ -0,0 +1,30 @@
+| deployment.env                                  | The environment variables for the pod                                                                                                                   | No                | See values.yaml                                                                                                                                                                                                                                                             |
+| deployment.restartPolicy                        | The deployment restart policy                                                                                                                           | No                | Always                                                                                                                                                                                                                                                                      |
+| deployment.volumeMounts                         | The deployment volume mounts                                                                                                                            | No                | &emsp; - name: metastore-service-config <br/>&emsp;&emsp;mountPath: /opt/apache-hive-metastore-3.0.0-bin/conf/metastore-site.xml <br/>&emsp;&emsp;subPath: metastore-site.xml                                                                                               |
+| deployment.volumes                              | The deployment volumes                                                                                                                                  | No                | &emsp; - name: metastore-service-config <br/>&emsp;&emsp;configMap: <br/>&emsp;&emsp;&emsp;name: metastore-service-config <br/>&emsp;&emsp;&emsp;items: <br/>&emsp;&emsp;&emsp;&emsp; - key: metastore-site.xml <br/>&emsp;&emsp;&emsp;&emsp;&emsp;path: metastore-site.xml |
+| service.spec.ports                              | The service spec ports                                                                                                                                  | No                | &emsp; - name: "thrift" <br/>&emsp;&emsp;port: 9083 <br/>&emsp;&emsp;targetPort: 9083                                                                                                                                                                                       |
+| configMap.metastoreServiceConfig.baseProperties | Default configuration for the metastore service                                                                                                         | No                | See values.yaml                                                                                                                                                                                                                                                             |
```

Comment:
I: The original version of this README enumerated every property that was set in the default.  This resulted in an illegible line in the README source.  Opted to replace it with a referral to the source material, but open to discussion as to whether this is appropriate.

Reply:
I think this is fine, but you should be able to actually link values.yaml with the relative path.